src/org/labkey/test/components/glassLibrary/grids/QueryGrid.java

```diff
@@ -51,10 +58,58 @@ public WebDriver getDriver()
         return _driver;
     }
 
-    public ResponsiveGrid getGrid()
+    // get rowMaps
+
+    public Map<String, String> getRowMap(int rowIndex)
     {
-        _responsiveGrid = new ResponsiveGrid.ResponsiveGridFinder(_driver).find(_queryGridPanel);
-        return _responsiveGrid;
+        return getGrid().getRowMap(rowIndex);
+    }
+
+    public Map<String, String> getRowMap(String containsText)
```

Comment:
I think naming the parameter **contains**Text can be a little misleading. What is ultimately called is withText which does a "=" comparison. Using the word contains implies you can give a partial string.
For example look at our locators (withClassContaining), you can give a partial string for it to resolve. In that case the contains here means something different than the contains elsewhere in our code.